package-imports

```diff
@@ -6,79 +6,48 @@ apparmor
 apt-transport-https
 arptables
 auditd
-awscli
-azure-vm-utils
 binutils
-bird2
 bsdextrautils
 btrfs-progs
-bubblewrap
-build-essential
```

Comment:
used by Nvidia installer test container https://github.com/gardenlinux/gardenlinux-nvidia-installer/blob/5708d7e5a7b5d3642465f958e577ee8f0ceaea71/test.Dockerfile#L7 

not sure if we need this test.Dockerfile